src/page-components/Product/SearchItemResult.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react'
+import ProductItem from './ProductItem'
+import Pagination from '@mui/material/Pagination'
+import { useNavigate } from 'react-router-dom'
+import { path } from '../../constants/path'
+import useQuery from '../../hooks/useQuery'
+import ProductSkelete from './ProductSkeleton'
+import { useSelector } from 'react-redux'
+
+export default function SearchItemResult({ products }) {
+  const { products: productList, pagination } = products
+  const query = useQuery()
+  const navigate = useNavigate()
+  const [page, setPage] = useState(1)
+  const loading = useSelector(state => state.app.loading)
+
+  useEffect(() => {
+    const { page } = query
+    page && setPage(parseInt(page))
+  }, [query])
+
+  const handleChangePage = (event, value) => {
+    setPage(value)
+    navigate(path.home + `?page=${value}`)
+  }
+  return (
+    <>
+      {loading ? (
+        <ProductSkelete />
+      ) : (
+        productList && (
+          <div className="flex flex-wrap">
+            {productList.map(product => (
+              <ProductItem key={product._id} product={product} />
```

Comment:
Thay vì em truyền `product={product}`, xong rồi ở component ProductItem em lại gọi
```
function ProductItem({ product })
const { _id, image, name, sold, price, rating } = product
```
Thì em có thể truyền như này
```suggestion
              <ProductItem key={product._id} {...product} />
```
Lúc này component ProductItem sẽ là
`function ProductItem({ _id, image, name, sold, price, rating })`

Reply:
em đã fix